LEGO1/legoomni.h

```diff
@@ -21,6 +21,7 @@
 #include "legoplantmanager.h"
 
 class LegoSoundManager;
+class MxTransitionManager;
```

Comment:
Looks like this class might need a refactor in terms of includes and forward declarations, looks kind of messy right now.